### ISSUE TITLE
No http.route in Ktor framework

### ISSUE THREAD

**Issue by alaegin** [open-telemetry/opentelemetry-java-instrumentation#9144]
**Describe the bug**

Hello!

There is no `http.route` attribute in traces when I use Ktor server framework.
Is it supported or not yet implemented?

**Steps to reproduce**

Execute any request to ktor, there won't be `http.route` attribute. `http.target` exists btw

**What did you expect to see?**

I expect to see both `http.route` and `http.target.` in trace

**What did you see instead?**

A clear and concise description of what you saw instead.

**What version are you using?**

Open Telemetry JavaAgent 1.28.0
Open Telemetry Ktor 2.0 1.28.0-alpha

**Comment by laurit** [comment]
We don't test ktor instrumentation with javaagent. Probably the issue is that netty instrumentation creates the server span before ktor instrumentation which suppresses the ktor instrumentation, including the part that sets the route in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/03664f2d6b5eb268879261c68eb7453d343c1c5c/instrumentation/ktor/ktor-2.0/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v2_0/server/KtorServerTracing.kt#L178 Instead of getting the context from `call.attributes` we could use `Context.current()` to ensure that this runs even when rest of the ktor instrumentation is suppressed.

**Comment by alaegin** [comment]
@laurit You're completely right! I tried to disable netty auto instrumentation and ktor plug-in works like a charm after that.
